plugins/nginx-vhosts/functions

```diff
@@ -6,14 +6,29 @@ source "$PLUGIN_AVAILABLE_PATH/config/functions"
 source "$PLUGIN_AVAILABLE_PATH/domains/functions"
 source "$PLUGIN_AVAILABLE_PATH/ps/functions"
 
+get_nginx_location() {
+  declare desc="check that nginx is at the expected location and return it"
+  local NGINX_LOCATION="/usr/sbin/nginx"
```

Comment:
This breaks it for people who package their own nginx. Why isn't `nginx` in the `$PATH` for you? It is for me.

Reply:
Well it was already broken for them, then. In the function `validate_nginx`, the sbin path is hardcoded in mainline, which means their config can never be valid.

Reply:
Actually not really. I have nginx installed in `/usr/local/bin` - custom compiled from openresty - and it works fine. Hardcoding to the `sbin` path also invalidates any special `$PATH` setting.

Reply:
Are you sure? If you run `which nginx` as the `dokku` user inside the Vagrant box, it does find it? I checked both on my server and in a Vagrant box, and that was not the case for me.

This is because `/sbin` and `/usr/sbin` are not in the path of regular user (not administrators) by default in Debian, AFAIK.

Reply:
It's hardcoded upstream right here:
https://github.com/dokku/dokku/blob/3a81c8f279ff823cde1ae7ce0ac4af994d1fd981/plugins/nginx-vhosts/functions#L12

As for the custom path, I think the best option here would  be for `get_nginx_location` both search in path with `which` and check the "standard" location even if it's not in path as a backup. Thoughts?

Reply:
Fancy, I guess I must also symlink it.

We should probably check the `$PATH` first and then the `/usr/sbin/nginx` location.